[rospy.client][INFO] 2021-01-18 10:25:14,535: init_node, name[/train_6713_1610965514505], pid[6713]
[xmlrpc][INFO] 2021-01-18 10:25:14,536: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-01-18 10:25:14,537: Started XML-RPC server [http://192.168.0.191:33379/]
[rospy.init][INFO] 2021-01-18 10:25:14,537: ROS Slave URI: [http://192.168.0.191:33379/]
[rospy.impl.masterslave][INFO] 2021-01-18 10:25:14,538: _ready: http://192.168.0.191:33379/
[rospy.registration][INFO] 2021-01-18 10:25:14,539: Registering with master node http://192.168.0.126:11311
[xmlrpc][INFO] 2021-01-18 10:25:14,539: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-01-18 10:25:14,638: registered with master
[rospy.rosout][INFO] 2021-01-18 10:25:14,638: initializing /rosout core topic
[rospy.rosout][INFO] 2021-01-18 10:25:14,644: connected to core topic /rosout
[rospy.simtime][INFO] 2021-01-18 10:25:14,646: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2021-01-18 10:25:14,665: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.126:34427/], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,670: topic[/odom] adding connection to [http://192.168.0.126:44411/], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,675: topic[/start_goal_points] adding connection to [http://192.168.0.126:40887/], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,680: topic[/scan] adding connection to [http://192.168.0.126:44411/], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,682: topic[/scan] adding connection to [http://192.168.0.126:46009/], count 1
[rospy.internal][INFO] 2021-01-18 10:25:14,686: topic[/is_start_episode] adding connection to [http://192.168.0.126:40755/], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,898: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,899: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.internal][INFO] 2021-01-18 10:25:14,901: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.core][INFO] 2021-01-18 10:25:23,541: signal_shutdown [atexit]
[rospy.internal][INFO] 2021-01-18 10:25:23,545: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-01-18 10:25:23,545: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2021-01-18 10:25:23,546: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2021-01-18 10:25:23,546: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.126:34427/
[rospy.internal][INFO] 2021-01-18 10:25:23,546: topic[/odom] removing connection to http://192.168.0.126:44411/
[rospy.internal][INFO] 2021-01-18 10:25:23,546: topic[/start_goal_points] removing connection to http://192.168.0.126:40887/
[rospy.internal][INFO] 2021-01-18 10:25:23,547: topic[/scan] removing connection to http://192.168.0.126:44411/
[rospy.internal][INFO] 2021-01-18 10:25:23,547: topic[/scan] removing connection to http://192.168.0.126:46009/
[rospy.internal][INFO] 2021-01-18 10:25:23,548: topic[/is_start_episode] removing connection to http://192.168.0.126:40755/
[rospy.impl.masterslave][INFO] 2021-01-18 10:25:23,548: atexit
[rospy.internal][WARNING] 2021-01-18 10:25:23,549: Unknown error initiating TCP/IP socket to 192.168.0.126:43043 (http://192.168.0.126:44411/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2021-01-18 10:25:23,550: Unknown error initiating TCP/IP socket to 192.168.0.126:50861 (http://192.168.0.126:40755/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

